datashuttle/datashuttle.py

```diff
@@ -969,15 +972,24 @@ def show_existing_projects(self):
         )
 
     @check_configs_set
-    def validate_project(self):
+    def validate_project(
```

Comment:
btw, the docstring here is also out of date. Now validation does much more

Reply:
For the CLI equivalent of this function, it would be nice to print "All good" at the end. As it is right now, running validate-project on a valid project from the CLI returns blank

Reply:
Thanks good catch, I have updated the docstring and raised an issue